### DISCUSSION TITLE
Substituir um termo dentro da tabela

### DISCUSSION THREAD

**Discussion by jornalistaCarlosOsorio** [sqlitebrowser/sqlitebrowser#2697] [Q&A]
Estou com dificuldades para substituir um termo dentro da tabela. Trocar um nome por exemplo. Modificar o topo da tabela é relativamente fácil. Acredito que o termo na tabela também. Contudo desconheço o caminho. 

Alguém poderia me ajudar?

Obrigado,

**Comment by chrisjlocke** [comment]
Não tenho certeza do que você quer dizer com 'termo'. Você pode fornecer uma captura de tela do que deseja alterar? Os nomes e tipos de coluna podem ser alterados, renomeados ou removidos. Os valores nos campos podem ser alterados. Não tenho certeza do que você está tentando mudar.
Fico feliz em ajudar quando eu entender.

---

I'm unsure what you mean by 'term'.  Can you provide a screenshot of what you want to change?  Column names and types can be changed, renamed or removed.  Values in fields can be changed.  I'm not sure what you're trying to change.
Happy to help once I understand though.

---

**Reply by jornalistaCarlosOsorio** [reply]
![image](https://user-images.githubusercontent.com/76069092/117873577-53fe6c80-b276-11eb-9dfe-696af1cb0282.png)

**Reply by jornalistaCarlosOsorio** [reply]
Obrigado pela ajuda! 

Quero trocar este nome: RENATO COSTA DE OLIVEIRA, mas não estou sabendo.

Obrigado,

**Reply by chrisjlocke** [reply]
Você clicou na célula uma vez para destacá-la - clique nela novamente e ela entrará no modo de edição. Digite o texto desejado e pressione Enter. Clique em 'Gravar alterações' na barra de ferramentas para salvar as alterações no banco de dados.

---

You've clicked the cell once to highlight it - click it again and it'll go into edit mode.  Type the text you want and press Enter.  Click 'Write changes' in the toolbar to save the changes to the database.

**Reply by jornalistaCarlosOsorio** [reply]
Sim, mas gostaria de trocar em todas as linhas da tabela com apenas uma operação. Essa tabela tem 18.185 linhas. Manualmente é impossível. Por isso, estou pedindo ajuda. Então, querida saber se dá para colocar um comando (código) para realizar essa operação.

Obrigado,
